examples/bevy_2048/src/movement/mod.rs

```diff
@@ -205,10 +207,19 @@ impl Output for Serial {
     }
 }
 
-#[derive(Debug, serde::Deserialize)]
+// #[derive(Debug, serde::Deserialize)]
 // {"type_name":"&st\nr","channel":2,"string":"unknown"}
+// struct Message<'a> {
+//     type_name: Cow<'a, str>,
+//     channel: usize,
+//     string: Cow<'a, str>,
+// }
+// {"type_name":"&str","channel":2,"elements":["up"],"dimensions":[1]}
+#[derive(Debug, serde::Deserialize)]
 struct Message<'a> {
     type_name: Cow<'a, str>,
     channel: usize,
-    string: Cow<'a, str>,
-}
+    elements: Cow<'a, [str; 1]>,
```

Comment:
@Michael-F-Bryan I'm unable to figure out what to change for the game to read the element output from the rune.